clients/cobol-lsp-vscode-extension/src/__tests__/services/copybook/CopybookURITest.spec.ts

```diff
@@ -41,7 +42,13 @@ describe("CopybooksPathGenerator tests", () => {
   });
   it("creates dataset path", () => {
     expect(CopybookURI.createDatasetPath(profile, dataset)).toEqual(
-      path.join("/projects", ".c4z", ".copybooks", "profile", "dataset"),
+      path.join(
+        Utils.getExtensionFolder(),
+        ZOWE_FOLDER,
```

Comment:
I'd prefer the `.zowe` literal here and no use of `Utils.getExtensionFolder`

Reply:
partly resolved at : 19c31007cf0f1c59dafd17843eeb8d6bae0c4e67

why not use of Utils.getExtensionFolder ?

Reply:
resolved at : 652cb705445212aca0650c9faf4fa53e2b0c4a48